src/Marketplace/MarketplaceScreen.js

```diff
@@ -65,19 +66,22 @@ export default function MarketplaceScreen({ navigation }) {
   const [produceList, setProduceList] = useState([]);
 
   const [filterVisibility, setFilterVisibility] = useState(false);
-
-  // const checkFavorites = () => {
-
-  // };
+  const [favoritesFilter, setFavoritesFilter] = useState(false);
 
   const getProduce = async () => {
+    let favorites = [];
+    await base('Users').find(userId, (err, record) => {
+      if (err) {
+        console.error(err);
```

Comment:
can you replace all console.errors in all of the screens w/         Alert.alert(err.error, err.message);